### DISCUSSION TITLE
Support For PS3 Theme Files

### DISCUSSION THREAD

**Discussion by AkiraKusanagi** [EmiyaSyahriel/CrossLauncher#53] [Ideas]
I was checking out some themes I had for the PS3 and I thought it would be cool if Cross Launcher supported the files so the launcher could be further customized. Like if you loaded a theme for the PS3 (From the file) it could customize both the icons and background of the launcher to look something like this:
![picture](https://github.com/EmiyaSyahriel/CrossLauncher/assets/137284132/1b7ec8a5-eb19-4400-858a-507b6d1d84fd)

**Comment by EmiyaSyahriel** [comment]
Based on this article : [PlayStation 3 Theme - PS3 Developer wiki](https://www.psdevwiki.com/ps3/PlayStation_3_Theme_(P3T))
PlayStation 3 Theme uses proprietary container based on binarized XML file, although the file inside is common file formats like ".PNG". 

I certainly wanted to implement this too, but, I don't think implementing the theme container reader right into the launcher is a simple task, given that the theme fille sometimes is also compressed.

I might not be able to implement this feature right away (or maybe it's sometime in the future?), but, I am open for anyone to make a pull request regarding this feature.
